package.json

```diff
@@ -83,11 +83,10 @@
     "@cypress/react": "5.12.4",
     "@cypress/webpack-dev-server": "1.8.3",
     "@fluentui/react-icons": "^2.0.166-rc.3",
-    "@griffel/babel-preset": "1.2.0",
-    "@griffel/jest-serializer": "1.0.3",
-    "@griffel/react": "1.0.3",
-    "@griffel/webpack-loader": "2.0.3",
-    "@linaria/babel-preset": "3.0.0-beta.14",
```

Comment:
what happened with this dependency?

Reply:
_I was sure that I commented it, but not..._

Before we have weird issues because of how hoisting works in `yarn`, but it's not needed anymore because of https://github.com/microsoft/griffel/pull/94.